index.html

```diff
@@ -37,6 +32,119 @@ <h3>I’am  web a developer.</h3>
             <li><img src="images/medium.png" alt="social-icon" /></li>
       </ul>   
     </section>
+       <section class="my-works1">
+
+
+        <h1>My Recent works</h1>
+
+     
+        <h3>Multi-Post Stories</h3>
+        <a class="img1"><img src="images/placeholder.png" alt="hamburger Icon" /></a>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        
+        
+          <ul class="lenguajes">
+            <li>css</li>
+            <li>html</li>
+            <li>boostrap</li>
+            <li>ruby</li>
+           
+          </ul>
+          
+          
+        <button type="submit">See Project</button>
+
+        </section>
+
+        <section class="my-works">
+        <div class="center">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="lenguajes">
+            <li>css</li>
+            <li>html</li>
+            <li>boostrap</li>
+            <li>ruby</li>
+           
+          </ul>
+          
+          <button type="submit">See Project</button>
+        </div>
+        
+      </section>
+
+      <section class="my-works">
+        <div class="center">
+          <h3>Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="lenguajes">
+            <li>css</li>
+            <li>html</li>
+            <li>boostrap</li>
+            <li>ruby</li>
+           
+          </ul>
+          
+          <button type="submit">See Project</button>
+        </div>
+        
+      </section>
+      
+      <section class="content">
+        <div class="about">
+          <h1>About me</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+             <a href="#">Get My Resume</a>
```

Comment:
- [ ] [Optional] This button like all other buttons present in your portfolio has to obey the Figma template design, I strongly suggest to add this style according to your Figma template

![image](https://user-images.githubusercontent.com/52635378/164943786-3ff119eb-d496-4847-bf3a-03fd56d0ce50.png)